addons/ai/XEH_preInit.sqf

```diff
@@ -4,4 +4,19 @@ ADDON = false;
 
 #include "XEH_PREP.hpp"
 
+if (isServer) then {
+    [QGVAR(updateActiveGroups), {
+        params ["_newGroups"];
+        private _spawnedGroups = GVAR(activeGroups) getOrDefault [_id, []];
+        _spawnedGroups append _newGroups;
+        GVAR(activeGroups) set [_id, _spawnedGroups];
```

Comment:
Just tested, you can spawn in saved groups endlessly, which means a wrong trigger can spawn the same id multiple times, clear `GVAR(savedGroups)` of the spawned group